app/api/__init__.py

```diff
@@ -297,6 +302,20 @@
 api.route(MicrolocationRelationshipRequired, 'microlocation_event',
           '/microlocations/<int:id>/relationships/event')
 
+# sessions
+api.route(EventChatMessageListAdmin, 'event_chat_message_list_admin',
+          '/admin/event-chat-messages')
```

Comment:
Why this ? Why can't you just check if the user is the admin or not programmatically and return data accordingly. We do similar things everywhere else. For eg. event list.

Adding non-standard endpoints like this make consuming these extremely difficult in the frontend.